### DISCUSSION TITLE
Introducing the New OP-ORDER Generation Feature in FortiPath

### DISCUSSION THREAD

**Discussion by cywf** [cywf/FortiPath#40] [General]
# Hello FortiPath Community

We're excited to announce a significant update to FortiPath that we believe will enhance your operational efficiency and security planning capabilities. With the latest addition of the OP-ORDER generation feature, our goal is to empower agents, operators, and users in managing their operations with greater precision and security.

## What's New?

- **OP-ORDER Generation**: A core update is the introduction of the OP-ORDER generation feature. This tool is designed to streamline the creation of operational documents, enabling agents to quickly generate tailored OP-ORDERS based on specific operational needs.

- **CryptPad Integration**: In our pursuit to offer secure and efficient operational planning tools, we've integrated CryptPad into FortiPath. CryptPad is an encrypted collaboration platform that offers various tools essential for security professionals. This integration means that users can now manage their operations with enhanced privacy and security.

- **Future Updates**: Looking ahead, we plan to enable users to deploy their own secure CryptPad server. This advancement is aimed at providing you with the autonomy to create a virtual Security Operations Center (vSOC) tailored to your unique operational requirements.

- **Comprehensive Toolset**: The update is part of our broader strategy to transform FortiPath into a comprehensive suite for security professionals. By leveraging technologies such as Terraform, Rust, ZeroTier, and Zapier, alongside the integration with OpenAI's Assistant API, we're enhancing FortiPath's capabilities in operational infrastructure development and planning.

## Getting Started with the OP-ORDER Feature

To utilize the new OP-ORDER generation feature, users can initiate the process through FortiPath's interface. The system will guide you through creating a customized OP-ORDER tailored to your operational information. This feature simplifies the planning process, allowing you to focus on the critical aspects of your operation.

### Your Feedback Matters

As we roll out this new feature, we're eager to hear your thoughts and feedback. Your insights are invaluable in helping us refine and enhance FortiPath to better serve the security community.

Join us in exploring the possibilities this new feature brings to your operational planning and management. Together, let's redefine operational efficiency and security in the field.

Stay safe and efficient,
_The FortiPath Team_

**Comment by SamuraiActual** [comment]
tracking
